bandit/core/utils.py

```diff
@@ -6,6 +6,7 @@
 import logging
 import os.path
 import sys
+from _ast import Constant
```

Comment:
Why not use ast.Constant?

Reply:
You are right, it should be `ast`. Fixed.